theme/sass/components/_text-and-image-component.scss

```diff
@@ -0,0 +1,57 @@
+/* Text and image component */
+
+.text-and-image-component {
+    width: 60%;
+    margin: 0 auto;
+
+    .text-and-image-container {
+
+        .image-container {
+            margin-bottom: 10px;
+            display: block;
+
+            &:hover {
+                img {
+                    filter: grayscale(0%);
+                }
+            }
+
+            img {
+                width: 400px;
+                height: 200px;
+                object-fit: cover;
+                filter: grayscale(100%);
+                transition: all .4s ease-in-out;
+            }
+        }
+    }
+}
+
+
+.text-and-image-component {
+    &.left {
```

Comment:
Everything inside and including &.left { } can go up to line 6. and also do the same for &.right {} (put it after left).

Reply:
done